### ISSUE TITLE
Proxy Gust Mode Multi Sessions

### ISSUE THREAD

**Issue by B3clawed** [kdcz/3s_atc#48]
In guest mode when I use proxies and try doing 5 sessions of each then click add and start it only shoes one session for each proxy.

**Comment by kdcz** [comment]
Yeah I know I forgot to implement that It should be done for next release

**Comment by mthurston94** [comment]
Hi I manage to cop a paid of yeezys today with just a few task. Next release I want to run 200-300 task what server specs would you recommend?

**Comment by kdcz** [comment]
Idk for real lol its a huge amount maybe you'll need 2 servers

**Comment by mthurston94** [comment]
just curious what the most u ever ran at once?

**Comment by B3clawed** [comment]
Sweet, also if I use your custom browser it uses SOOO much system memory
and I have 16gb

**Comment by B3clawed** [comment]
Another thing, can I join your project? I could work on the GUI and make it
look pretty nice. Here's a screenshot of a similar program I have been
working on but isn't done yet..
[image: Inline image 1]

**Comment by kdcz** [comment]
@mrthueston94 don't really remember 30 maybe on a single PC,but you make some tests before to prevent your PC from crashing during the release

@B3clawed Its the same ram usage as Chrome.Thanks for the help but I don't give much attention to the GUI since its a free bot and not a commercial one but thanks

**Comment by B3clawed** [comment]
I could take like an hour and make it look 10x better though you should just let me do it XD No point in not getting some help
